js/App.js

```diff
@@ -110,6 +116,10 @@ export default class App extends Component {
   }
 }
 
+App.propTypes = {
+  dispatch: PropTypes.func,
```

Comment:
It's not clear to me where this `dispatch` prop is coming from.
Is it passed directly from `<AppProvider/>`? Does it come from Redux?